src/lib/elements/RadioButtonsGroup.svelte

```diff
@@ -2,9 +2,18 @@
 <script lang="ts">
 	import { createEventDispatcher } from "svelte";
 
+	/**
+	 * The content of each toggle. Likely a string or an icon.
+	 */
 	export let values: unknown[] = [];
+	/**
+	 * The index of the default toggle. Defaults to 0, set to 0 if the value is out of bounds of `values`.
+	 */
 	export let defaultIndex = 0;
```

Comment:
Make `defaultIndex` constant

Reply:
Nope! A prop can't be a `const`, it has to be a `let`.
I should however use the `$:` format to dynamically update the variable right below but it works fine as-is